CMakeLists.txt

```diff
@@ -9,13 +9,13 @@ set(CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "")
 project(rocm-cmake)
 
 install(DIRECTORY share DESTINATION .)
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/ROCMConfigVersion.cmake DESTINATION share/rocm/cmake)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/ROCMConfigVersion.cmake DESTINATION "share/rocm/cmake")
```

Comment:
Is the quotes really necessary?

Reply:
No, they're not. When removing the ROCMConfigVersion.cmake file, I had to add this line back and added the quotes out of habit. They serve no additional purpose.

Reply:
I see.